include/alpaka/math/MathGenericSycl.hpp

```diff
@@ -0,0 +1,78 @@
+/* Copyright 2020 Jan Stephan
+ *
+ * This file is part of Alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#ifdef ALPAKA_ACC_SYCL_ENABLED
+
+#    include <alpaka/math/abs/AbsGenericSycl.hpp>
+#    include <alpaka/math/acos/AcosGenericSycl.hpp>
+#    include <alpaka/math/asin/AsinGenericSycl.hpp>
+#    include <alpaka/math/atan/AtanGenericSycl.hpp>
+#    include <alpaka/math/atan2/Atan2GenericSycl.hpp>
+#    include <alpaka/math/cbrt/CbrtGenericSycl.hpp>
+#    include <alpaka/math/ceil/CeilGenericSycl.hpp>
+#    include <alpaka/math/cos/CosGenericSycl.hpp>
+#    include <alpaka/math/erf/ErfGenericSycl.hpp>
+#    include <alpaka/math/exp/ExpGenericSycl.hpp>
+#    include <alpaka/math/floor/FloorGenericSycl.hpp>
+#    include <alpaka/math/fmod/FmodGenericSycl.hpp>
+#    include <alpaka/math/isfinite/IsfiniteGenericSycl.hpp>
+#    include <alpaka/math/isinf/IsinfGenericSycl.hpp>
+#    include <alpaka/math/isnan/IsnanGenericSycl.hpp>
+#    include <alpaka/math/log/LogGenericSycl.hpp>
+#    include <alpaka/math/max/MaxGenericSycl.hpp>
+#    include <alpaka/math/min/MinGenericSycl.hpp>
+#    include <alpaka/math/pow/PowGenericSycl.hpp>
+#    include <alpaka/math/remainder/RemainderGenericSycl.hpp>
+#    include <alpaka/math/round/RoundGenericSycl.hpp>
+#    include <alpaka/math/rsqrt/RsqrtGenericSycl.hpp>
+#    include <alpaka/math/sin/SinGenericSycl.hpp>
+#    include <alpaka/math/sincos/SinCosGenericSycl.hpp>
+#    include <alpaka/math/sqrt/SqrtGenericSycl.hpp>
+#    include <alpaka/math/tan/TanGenericSycl.hpp>
+#    include <alpaka/math/trunc/TruncGenericSycl.hpp>
```

Comment:
Could you please merge the SYCL math function implementations into a single file like it was done recently on the develop branch?